parsec/api/data/entry.py

```diff
@@ -34,7 +34,7 @@ class EntryID(UUID4):
 
 class EntryName(StrBased):
     # Ignore the REGEX
-    REGEX = None
+    REGEX = re.compile(".*")
```

Comment:
Should modify typing for `StrBased` to `Optional[Pattern[str]]` instead

Reply:
Fixed